default.nix

```diff
@@ -4,6 +4,7 @@ in
 { system ? builtins.currentSystem
 , config ? {}
 , dconfig ? "testnet_staging"
+, gitrev ? "unknown"
```

Comment:
@cleverca22 @domenkozar can I ask you guys to commit this change to branch? In principle I only need `GITREV` env variable appropriately set